classes/sota.bbclass

```diff
@@ -17,7 +17,7 @@ EXTRA_IMAGEDEPENDS_append_sota = " parted-native mtools-native dosfstools-native
 # Please redefine OSTREE_REPO in order to have a persistent OSTree repo
 OSTREE_REPO ?= "${DEPLOY_DIR_IMAGE}/ostree_repo"
 # For UPTANE operation, OSTREE_BRANCHNAME must start with "${MACHINE}-"
-OSTREE_BRANCHNAME ?= "${MACHINE}-ota"
+OSTREE_BRANCHNAME ?= "${MACHINE}"
```

Comment:
Perhaps a rather minor matter, but the comment there claims OSTREE_BRANCHNAME should start with "${MACHINE}-", but now the default is just "${MACHINE}" without the dash. Does that matter at all?

Reply:
The comment is probably obsolete, current CI just uses branch name as hardware ID.

Reply:
That sounds about right. I hadn't seen that Uptane restriction enforced anywhere. Is there any need at all for the branch name to include MACHINE at all, then, or is it just a loose convention? If so, we should just remove the comment (not that you need to necessarily do that in this PR).